packages/medusa/src/loaders/custom-api.ts

```diff
@@ -0,0 +1,56 @@
+import { Express, NextFunction, Request, Response, Router } from "express"
+import { ConfigModule } from "../types/global"
+
+enum AllowedMethods {
+  GET = "GET",
+  POST = "POST",
+  DELETE = "DELETE",
+}
+
+type Handler = (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => Promise<void>
+
+export type MedusaRoute = {
+  name: string
+  method: AllowedMethods
+  path: string
+  handler: Handler | Handler[]
```

Comment:
suggestion: I think it should be `handlers` and we could always expect an array of one or more handlers?